draft-ietf-tls-tls13.md

```diff
@@ -4949,6 +4980,12 @@ server is not able to guarantee full uniqueness of the handshake
 state. See {{early-data-indication}} for one mechanism to limit
 the exposure to replay.
 
+KCI resistance describes the security guarantees that a party may still have
+after its long-term secret has been compromised. TLS does not provide security
+for data communicated after the peer's long-term secret (signature key or
+external PSK) is compromised. It therefore does not provide post-compromise
+security {{CCG16}}, sometimes also referred to as backwards or future security.
```

Comment:
Re 1: my emphasis was unclear---I meant to contrast KCI resistance (if _your_ key has been compromised) with PCS (if your _peer's_ key has been compromised). Let me reword it a bit.